packs/data/feats.db/hijack-undead.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_id": "1HvSkbUjqrIMXLiY",
+    "data": {
+        "actionCategory": {
+            "value": ""
+        },
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "description": {
+            "value": "<p><strong>Frequency</strong> once per hour</p>\n<hr />\n<p>Your clockwork devices can commandeer the bodies of undead, bending them to your will. Make a melee Strike against an adjacent undead creature. On a success, instead of taking damage, the target becomes controlled by you if its level is equal to or lower than your level - 3. It can attempt a @Check[type:will|dc:resolve(@actor.attributes.classOrSpellDC.value)] saving throw against the higher of your class DC or spell DC to resist being @Compendium[pf2e.conditionitems.Controlled]{Controlled} by you. If the target is already under someone else's command, the controlling creature also rolls a saving throw, and the undead uses the better result.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected and temporarily immune for 24 hours.</p>\n<p><strong>Success</strong> The target is unaffected.</p>\n<p><strong>Failure</strong> The undead creature becomes a minion under your control for 1 minute. The control ends if you or an ally attacks the minion undead.</p>\n<p><strong>Critical Failure</strong> As failure, but the duration is 1 hour.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Frequency</strong> once per hour</p>\n<hr />\n<p>Your clockwork devices can commandeer the bodies of undead, bending them to your will. Make a melee Strike against an adjacent undead creature. On a success, instead of taking damage, the target becomes @Compendium[pf2e.conditionitems.Controlled]{Controlled by you if its level is equal to or lower than your level - 3. It can attempt a @Check[type:will|dc:resolve(@actor.attributes.classOrSpellDC.value)] saving throw against the higher of your class DC or spell DC to resist being controlled by you. If the target is already under someone else's command, the controlling creature also rolls a saving throw, and the undead uses the better result.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected and temporarily immune for 24 hours.</p>\n<p><strong>Success</strong> The target is unaffected.</p>\n<p><strong>Failure</strong> The undead creature becomes a minion under your control for 1 minute. The control ends if you or an ally attacks the minion undead.</p>\n<p><strong>Critical Failure</strong> As failure, but the duration is 1 hour.</p>"
```

Move reference to controlled up to the first instance